pyproject.toml

```diff
@@ -20,5 +23,8 @@ dev = [
     "pytest>=8.1.1",
     "pytest-cov>=5.0.0",
     "pre-commit>=3.6.1",
-    "nox>=2024.3.2"
+    "nox>=2024.3.2",
+]
+build = [
+    "hatch>=1.9.2"
```

Comment:
I have not seen this specification, why do we need to put the specific hatch version?

Reply:
This is under `optional-dependencies` and is not required. 
I have added it here, just in case someone wants to try and make a hatch-build locally. Ref: https://hatch.pypa.io/1.12/build/#building

Reply:
Let's remove that. We will be using the regular python build to build the package.

Reply:
Done.